src/lib/Components/Inbox/Conversations/ConversationSnippet.tsx

```diff
@@ -178,14 +204,25 @@ export default Relay.createContainer(ConversationSnippet, {
         }
         last_message
         last_message_at
-        artworks {
-          id
-          href
+        items {
           title
-          date
-          artist_names
-          image {
-            url
+          item {
+            __typename
+            ... on Artwork {
+              id
```

Comment:
Ah I see. Relay automatically adds the global ID field (which is `__id` instead of `id` btw) to the query, so normally there wouldn’t be a need to add it manually unless you are using it in your own code (e.g. for a `key` in a list of components).

The reason we needed to manually include the `__id` field in mutations is simply because our Relay fork was incomplete.